consensus/ibft/ibft_test.go

```diff
@@ -660,38 +659,49 @@ func (s *mockSyncer) Broadcast(b *types.Block) {
 
 type mockTxPool struct {
 	transactions          []*types.Transaction
+	demoted               []*types.Transaction
 	nonceDecreased        map[*types.Transaction]bool
 	resetWithHeaderCalled bool
 	resetWithHeadersParam []*types.Header
 }
 
-func (p *mockTxPool) ResetWithHeaders(headers ...*types.Header) {
-	p.resetWithHeaderCalled = true
-	p.resetWithHeadersParam = headers
+func (p *mockTxPool) Prepare() {
+
```

Comment:
Maybe add an explicit return here?

Reply:
IDE would mark it as a redundant return statement.